src/screens/UserManagement/UserRevamp/NewUserInvitationForm.res

```diff
@@ -0,0 +1,202 @@
+let p1MediumTextClass = HSwitchUtils.getTextClass((P1, Medium))
+let p2MediumTextClass = HSwitchUtils.getTextClass((P2, Medium))
+let p3MediumTextClass = HSwitchUtils.getTextClass((P3, Medium))
+let p3RegularTextClass = HSwitchUtils.getTextClass((P3, Regular))
+
+module ModulePermissionRender = {
+  @react.component
+  let make = (~elem: UserManagementTypes.userModuleType, ~index, ~customCss="") => {
+    open UserUtils
+
+    let iconForAccess = access =>
+      switch access->stringToVariantMapperForAccess {
+      | View => "eye-outlined"
+      | Manage => "pencil-outlined"
+      }
+
+    <div key={index->Int.toString} className={`flex justify-between ${customCss}`}>
+      <div className="flex flex-col gap-2 basis-3/5 ">
+        <p className=p2MediumTextClass> {(elem.parentGroup :> string)->React.string} </p>
+        <p className=p3RegularTextClass> {elem.description->React.string} </p>
+      </div>
+      <div className="flex gap-2 h-fit">
+        {elem.groups
+        ->Array.map(item => {
+          <p
+            className={`py-0.5 px-2 rounded-full bg-gray-200 ${p3RegularTextClass} flex gap-1 items-center`}>
+            <Icon name={item->iconForAccess} size=12 />
+            <span> {(item :> string)->React.string} </span>
+          </p>
+        })
+        ->React.array}
+      </div>
+    </div>
+  }
+}
+module RoleToPermission = {
+  @react.component
+  let make = (~roleInfo: array<UserManagementTypes.userModuleType>, ~roleDict, ~role) => {
+    open LogicUtils
+    let userAcessGroup = roleDict->getDictfromDict(role)->getStrArrayFromDict("groups", [])
+    let (modulesWithAccess, moduleWithoutAccess) = UserUtils.modulesWithUserAccess(
+      roleInfo,
+      userAcessGroup,
+    )
+
+    <div className="flex flex-col gap-8">
+      {modulesWithAccess
+      ->Array.mapWithIndex((elem, index) => {
+        <ModulePermissionRender elem index />
+      })
+      ->React.array}
+      {moduleWithoutAccess
+      ->Array.mapWithIndex((elem, index) => {
+        <ModulePermissionRender elem index customCss="text-grey-200" />
+      })
+      ->React.array}
+    </div>
+  }
+}
+
+module NoteComponent = {
+  @react.component
+  let make = () => {
+    <div className="flex gap-2 items-start justify-start">
+      <Icon name="info-vacent" size=18 className="" customIconColor="!text-gray-400" />
+      // TODO : Change based on selection
+      <span className={`${p3RegularTextClass} text-gray-500`}>
+        {"You can only invite people for 'Hyperswitch US' here. To invite users to another organisation, please switch the organisation."->React.string}
```

Comment:
Please create a issue for this or you might forgot

Reply:
Updated this based on entity